### ISSUE TITLE
Periodogram not using user-input frequencies

### ISSUE THREAD

**Issue by jkcarlberg** [KeplerGO/lightkurve#212]
#### Problem description

The documentation for light curve's periodogram says that you can input an array of frequencies for which you would like the power spectrum calculated. However, when I try to do so it still computes the power spectrum for the default frequency array.

#### Example

I am interested in frequencies lower than 1 microHertz. In the snippet below, "lc" is a lightcurve built up from multiple quarters of KIC 6948128.
```python
per = lc.periodogram() #periodogram on default frequency array

#Define a new frequency array
freq_low = 1./((lc.time[-1]-lc.time[0])/2*u.day) #Define new low frequency cut-off 
freq_low2 = freq_low.to(u.uHz)  #Convert to microHertz
freq_high = np.max(per.frequencies)  #Use default high frequency cut-off
freq = np.linspace(freq_low2.value, freq_high.value,num=int(len(per.frequencies)/2)) #Create full  array

print(np.min(freq)*u.microhertz)

> 0.016227840520383976 uHz

per2=lc.periodogram(frequencies=freq*u.microhertz)
print(np.min(per2.frequencies))

> 1.0 uHz

```

#### Expected behavior

The frequencies in `per2.frequencies`  should match the input `freq`, but do not.  I have tried both with explicitly defining `freq` as a quantity (as shown here) and defining `freq` as a regular numpy array.

#### Environment:
- platform (e.g. Linux, OSX, Windows): OSX
- lightkurve version (e.g. 1.0b6): 1.0b12
- installation method (e.g. pip, conda, source): pip

**Comment by barentsen** [comment]
@jkcarlberg Many thanks for reporting this issue!  :+1: :+1: :+1:  (We did rush this feature ahead of KASC last month!)

I implemented a bug fix (#213) which I expect to merge later today.  Stay tuned.

**Comment by barentsen** [comment]
Thanks again @jkcarlberg!

The bugfix has been merged and a new version of lightkurve, v1.0b13, has been released that includes the fix.  You can install it using `pip install lightkurve --upgrade`.

Let us know if you encounter any other issues!  Also keep an eye out for future improvements to the `Periodogram` class which will be merged as part of #189.
